src/main/java/com/compilercharisma/chameleonbusinessstudio/exception/NoUserLoggedInHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.compilercharisma.chameleonbusinessstudio.exception;
```

Comment:
I think its appropiate this class should be moved to the `controller` package because it returns HTTP statuses in case of errors, very minor though.